### PR TITLE
blog post for 1.42 (guaranteed end to end encryption)

### PR DESCRIPTION
**Pull request by hpk42**
This post is not trying to re-explain everything we have in FAQ and securejoin docs
but rather provide a somewhat splashy "news" item. 

Could use some help with better screenshots (and icons) or related formatting/styling issues.  Also the green-checkmark as "post image" in the blog list is badly cropped.

### CODE REVIEW

File: _posts/2023-11-20-jumbo-42.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+[new SecureJoin protocols documentation site](https://securejoin.delta.chat)
+which also serves to prepare an independent security audit scheduled for 2024. 
+
+## A lot of other good news (and submit the bugs, please!)
+
+The 1.42 releases also contain many other improvements, among them:
+
+- 🔄 Accept/Blocked, Archived, Pinned, Mute is synced across all your devices,
+
+- 🗜️ Images are now compressed (unless you send them as files),
```
Comment (adbenitez) [inline _posts/2023-11-20-jumbo-42.md]: this was the case already for android, is this about desktop?

File: _posts/2023-11-20-jumbo-42.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+- 🖼️ Global Gallery with your pictures, documents, media across all chats,
```
Comment (adbenitez) [inline _posts/2023-11-20-jumbo-42.md]: "on desktop"

Comment (hpk42) [reply _posts/2023-11-20-jumbo-42.md]: There is this "suggest" feature of github btw ;)

File: _posts/2023-11-20-jumbo-42.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  many of which are operating in surveillance oriented environments
+  or are otherwise exposed to capable adversaries. 
+
+- **Surprises because of unencrypted messages**: 
+  Users can encounter an irritating lack of end-to-end encryption 
+  when they send a first message or 
+  if their chat partners don't consistently use Autocrypt-capable apps.
+  They would send an un-encrypted message and only notice it after they sent a message. 
+  In practise, this is less of a problem than many experts think 
+  but no question: it is a valid concern.
```
Comment (missytake) [inline _posts/2023-11-20-jumbo-42.md]:
```suggestion
  but there is no question: it is a valid concern. 
```

in english this is less of an idiom than in german.

File: _posts/2023-11-20-jumbo-42.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+Read more in our [revised Encryption and Security FAQ](https://delta.chat/en/help#encryption-and-security) 
+which is also contained and shipped with all Delta Chat apps,
+and also discusses security-relevant topics such as OpenPGP security, 
+Metadata protection and ephemeral messaging. 
+
+Lastly, you may read a lot more technical and scientific background on our 
+[new SecureJoin protocols documentation site](https://securejoin.delta.chat)
+which also serves to prepare an independent security audit scheduled for 2024. 
+
+## A lot of other good news (and submit the bugs, please!)
```
Comment (missytake) [inline _posts/2023-11-20-jumbo-42.md]: That's bold - users might think "oh, so this software has bugs? Maybe I wait for the finished version then.

```suggestion
## A lot of other good news
```

File: _posts/2023-11-20-jumbo-42.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
+---
+title: Guaranteed End-to-End encryption and other good news
+author: holga
+image: ../assets/home/green-checkmark.svg
```
Comment (missytake) [inline _posts/2023-11-20-jumbo-42.md]: On https://staging.delta.chat/757/en/blog, the image is slightly cropped... maybe this is better?

```suggestion
image: ../assets/blog/e2ee-activated.png
```

**Review by missytake — APPROVED** [review]
Sounds good! Thanks for the draft.

File: _posts/2023-11-20-jumbo-42.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+as well as [TeamUSEC](https://teamusec.de) 
+all helped us to streamline usability and implementation issues 
+for guaranteed end-to-end encryption. 
+
+
+## Autocrypt: convenient but "opportunistic" E2E security 
+
+<img src="../assets/logos/autocrypt.svg" width="140" style="float:right; margin-left:1em;" />  
+
+Let us recall two frequent criticism of Delta Chat's [Autocrypt](https://autocrypt.org)
```
Comment (adzialocha) [inline _posts/2023-11-20-jumbo-42.md]: Plural criticisms?

File: _posts/2023-11-20-jumbo-42.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+Let us recall two frequent criticism of Delta Chat's [Autocrypt](https://autocrypt.org) 
+end-to-end encryption and its *opportunistic* nature: 
+
+- [MITM attack](https://en.wikipedia.org/wiki/Man-in-the-middle_attack): 
+  Compromised infrastructure is arguably a stronger concern 
+  for Delta Chat than Signal or Whatsapp users.
+  After all, there is a not a single central 
+  but many hundreds of different e-mail providers used with Delta Chat today,
+  many of which are operating in surveillance oriented environments
+  or are otherwise exposed to capable adversaries.
```
Comment (adzialocha) [inline _posts/2023-11-20-jumbo-42.md]: I'm not sure I follow the text here, as someone's who's new to the autocrypt criticism. I know this is not to explain everything again but maybe you can refer to the criticism points (in the FAQ etc.)?

Also I'm not sure about the answer to mitm, for unencrypted initial messages you state your solution clearly, even with a nice example, but for mitm you talk about critical infrastructure, how does it relate to the criticism or a solution?

File: _posts/2023-11-20-jumbo-42.md (hunk 5 above)
Comment (r10s) [reply _posts/2023-11-20-jumbo-42.md]: i will do a fix now

File: _posts/2023-11-20-jumbo-42.md (hunk 1 above)
Comment (r10s) [reply _posts/2023-11-20-jumbo-42.md]:
```suggestion
- 🗜️ Images are now compressed also on desktop (unless you send them as files),
```

File: _posts/2023-11-20-jumbo-42.md (hunk 2 above)
Comment (r10s) [reply _posts/2023-11-20-jumbo-42.md]:
```suggestion
- 🖼️ All platforms now have a "Global Gallery" with your pictures, documents, media across all chats,
```

File: _posts/2023-11-20-jumbo-42.md (hunk 1 above)
Comment (hpk42) [reply _posts/2023-11-20-jumbo-42.md]: thanks, all applied/resolved, also added links to android/ios/desktop/core changelogs.

File: _posts/2023-11-20-jumbo-42.md (hunk 4 above)
Comment (hpk42) [reply _posts/2023-11-20-jumbo-42.md]: remove dit, thanks,

File: _posts/2023-11-20-jumbo-42.md (hunk 3 above)
Comment (hpk42) [reply _posts/2023-11-20-jumbo-42.md]: applied

File: _posts/2023-11-20-jumbo-42.md (hunk 7 above)
Comment (hpk42) [reply _posts/2023-11-20-jumbo-42.md]: i changed this paragraph. hopefully better now.

File: _posts/2023-11-20-jumbo-42.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+- 🔄 Accept/Blocked, Archived, Pinned, Mute actions are now synced across all your devices,
+
+- 🗜️ Images are now compressed also on Desktop (unless you send them as files), 
+
+- 🖼️ Global Gallery with your pictures, documents, media across all chats,
+  now also on Desktop 
+
+- ❤️  Reactions can be enabled in Android advanced settings 
+
+- 🐞 Bug fixes, bug fixes and more bug fixes (all dev areas got many issues
```
Comment (missytake) [inline _posts/2023-11-20-jumbo-42.md]:
```suggestion
- 🐞 Bug fixes, bug fixes and more bug fixes (in all dev areas, many issues got
```

**Review by missytake — APPROVED** [review]
Looks good!

**Review by r10s — APPROVED** [review]
excellent!

File: _posts/2023-11-20-jumbo-42.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+  resolved ) -- Android went from a dozen PRs 
+  and roughly [50 issues to 5!](https://github.com/deltachat/deltachat-android/issues)
```
Comment (r10s) [inline _posts/2023-11-20-jumbo-42.md]: ftr, as this changes forth and back:

i also find the current state ("50 issues to 5!") with concrete 10x numbers catchier - even if the numbers are not totally precise :)

**Pull request merged**